### ISSUE TITLE
Impossible to disable tray icon in notification area

### ISSUE THREAD

**Issue by teohhanhui** [flathub/com.slack.Slack#45]
"Leave app running in notification area when the window is closed" option has no effect.

> Choosing not to leave the app running will remove the Slack icon (and badges) from your notifications area.

https://get.slack.help/hc/en-us/articles/201355156-Guide-to-Slack-notifications-#linux-1

It works with https://aur.archlinux.org/packages/slack-desktop/

**Comment by FakeShemp** [comment]
It works as intended for me on Manjaro Gnome. Which distro are you using?

**Comment by teohhanhui** [comment]
Manjaro GNOME! :laughing:

**Comment by teohhanhui** [comment]
It's weird because it works with https://flathub.org/apps/details/org.telegram.desktop

but not Slack...

**Comment by teohhanhui** [comment]
To clarify: Closing the window quits the app as expected, but the tray icon is visible when the app is running, which should not be the case when the option is unchecked.

**Comment by FakeShemp** [comment]
I'm now on Ubuntu, but yes, I'm having the same issue. However, I'm having the same issue with the .deb file directly from Slack as well, so I'm thinking it's an upstream issue.

**Comment by laidbackware** [comment]
I'm using Pop OS and this issue appeared when upgrading from 19.10 to 20.04

**Comment by iraklisg** [comment]
I was able to solve this by installing the `libappindicator-gtk3` dependency as stated [here](https://aur.archlinux.org/packages/slack-desktop/#comment-778093)

**Comment by teohhanhui** [comment]
@iraklisg This bug is about *disabling* the tray icon.

**Comment by jimmyrutherford** [comment]
yeah its very annoying. I hate having pointleess stuff in my system tray and this is definately pointless

**Comment by evan-a-a** [comment]
Closing due to no actionable fixes for the flatpak.
